op-supervisor/supervisor/backend/backend.go

```diff
@@ -632,6 +634,8 @@ func (su *SupervisorBackend) FindSealedBlock(ctx context.Context, chainID eth.Ch
 func (su *SupervisorBackend) AllSafeDerivedAt(ctx context.Context, source eth.BlockID) (map[eth.ChainID]eth.BlockID, error) {
 	chains := su.cfgSet.Chains()
 	ret := map[eth.ChainID]eth.BlockID{}
+
+	// Note: no need to reorg/rewind lock: everything is derived from the same L1 block
```

Comment:
would it be worthwhile to still use the read handle so that future modifications don't invalidate this comment?